pkg/mgrconfig/config.go

```diff
@@ -96,9 +96,10 @@ type Config struct {
 	// Mailx is the only supported mailer. Please set it up prior to using this function.
 	EmailAddrs []string `json:"email_addrs,omitempty"`
 
-	DashboardClient string `json:"dashboard_client,omitempty"`
-	DashboardAddr   string `json:"dashboard_addr,omitempty"`
-	DashboardKey    string `json:"dashboard_key,omitempty"`
+	DashboardClient    string `json:"dashboard_client,omitempty"`
+	DashboardAddr      string `json:"dashboard_addr,omitempty"`
+	DashboardKey       string `json:"dashboard_key,omitempty"`
+	DashboardUserAgent string `json:"dashboard_user_agent,omitempty"`
```

Comment:
You'll use it for the dashboard connection but it is a manager User-Agent, not dashboad.

Reply:
Let's call it UserAgent?

Reply:
It is the User-Agent that the manager will send when connecting to the dashboard, since it is used in `dashapi.New`.

But I don't have strong reasons to disagree with calling it just `UserAgent`.